x-pack/plugins/monitoring/server/config.test.ts

```diff
@@ -3,14 +3,22 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+
+import fs from 'fs';
```

Comment:
It's probably just me who doesn't see the big picture, but the changes to this file doesn't seem to be related to the PR?

Reply:
Responded in https://github.com/elastic/kibana/pull/76518/files#r485774436